plane/source/docs/airspeed.rst

```diff
@@ -202,30 +201,30 @@ or loiter will achieve the required direction changes, cross-country
 flying will not. To enable automatic airspeed sensor calibration, set
 the value of ARSPD_AUTOCAL to 1.
 
-The user can assign the connection pin for the Air Speed Sensor
-===============================================================
+Using a different pin for the Airspeed Sensor

```

Comment:
I tend to use "sentence capitalisation" for headings (and "All words capitalised" for title). So this would be "airspeed sensor" in the heading.